constants.go

```diff
@@ -41,6 +41,9 @@ const (
 	// TracerHostnameTagKey used to report host name of the process.
 	TracerHostnameTagKey = "jaeger.hostname"
 
+	// TracerIPTagKey used to report ip of the process.
+	TracerIPTagKey = "ip"
```

Comment:
This is extremely generic, wdyt about setting it as "process.ip" and converting it to "ip" when converting to jaeger.thrift process?